.circleci/config.yml

```diff
@@ -0,0 +1,57 @@
+version: 2.1
+
+executors:
+  python38:
+    docker:
+      - image: circleci/python:3.8
```

Comment:
Does specifying the version of the image mean we won't have access to the other python versions when doing unit testing, etc?

Reply:
No, i'll configure different executors for each version of python we support.